src/Sulu/Bundle/SecurityBundle/Entity/User.php

```diff
@@ -244,4 +244,16 @@ public function getFullName()
         return null !== $this->getContact() ?
             $this->getContact()->getFullName() : $this->getUsername();
     }
+
+    /**
+     * @VirtualProperty
+     * @SerializedName("username")
+     * @Groups({"frontend", "fullUser"})
+     *
+     * @return string
+     */
+    public function getUsername()
+    {
+        return parent::getUsername();
```

Comment:
Why can't we add the annotations to the parent directly? Just calling the parent function and doing nothing else would be unnecessary otheriwse...